src/main/java/banking/dao/AccountDAO.java

```diff
@@ -0,0 +1,155 @@
+package banking.dao;
+
+import banking.configuration.DBManager;
+import banking.model.Account;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class AccountDAO {
+
+    private final static String INSERT_INTO = "INSERT INTO account (number, pin) VALUES (?, ?)";
+    private final static String SELECT_CARD_NUMBER = "SELECT count(number) FROM account WHERE number = (?)";
+    private final static String SELECT_ALL = "SELECT * FROM account";
+    private final static String SELECT_ID = "SELECT id FROM account WHERE number=? AND pin=?";
+    private final static String GET_CARD = "SELECT * FROM account WHERE id = ?";
+    private final static String UPD_BALANCE = "UPDATE account SET balance = (balance + ?) WHERE id = ?";
+    private final static String GET_MONEY = "UPDATE account SET balance = (balance + ?) WHERE number = ?";
+    private final static String SEND_MONEY = "UPDATE account SET balance = (balance - ?) WHERE id = ?";
+    private final static String DEL_ACC = "DELETE FROM account WHERE id = ?";
+    private DBManager dbManager;
+
+    public AccountDAO(DBManager dbManager) {
+        this.dbManager = dbManager;
+    }
+
+    public void insertCard(Account account) {
+        try (
+             PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(INSERT_INTO)) {
+            preparedStatement.setString(1, account.getCard().getNumber());
+            preparedStatement.setString(2, account.getCard().getPIN());
+            preparedStatement.executeUpdate();
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public int getId() {
+        int id = 0;
+        try (
+             Statement stmt = dbManager.getConnection().createStatement()) {
+
+            ResultSet res = stmt.executeQuery(SELECT_ALL);
+            while (res.next()) {
+                id = res.getInt("id");
+            }
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+        return id;
+    }
+
+    public boolean containCard(String cardNumber) {
+        ResultSet res;
```

Comment:
Why do you need to create `ResultSet ` here, if use it only in the `try-catch` block?

Reply:
fixed